internal/databases/postgres/tar_interpreter.go

```diff
@@ -98,6 +98,13 @@ func (tarInterpreter *FileTarInterpreter) Interpret(fileReader io.Reader, fileIn
 	tracelog.DebugLogger.Println("Interpreting: ", fileInfo.Name)
 	targetPath := path.Join(tarInterpreter.DBDataDirectory, fileInfo.Name)
 	fsync := !viper.GetBool(internal.TarDisableFsyncSetting)
+	if fileInfo.Typeflag != tar.TypeDir {
+		if err := os.Remove(targetPath); err == nil {
```

Comment:
We can't remove the existing files during the incremental backup restore because it will break the delta backups restoration. WAL-G does the page-level increments, it does not overwrite the entire file during the delta backups unpacking.

Reply:
Clear, thanks...